modin/pandas/test/test_io.py

```diff
@@ -2491,6 +2493,20 @@ def test_to_gbq(self):
         ):
             modin_df.to_gbq("modin.table")
 
+    def test_read_gbq_mock(self):
+        test_args = ("fake_query",)
+        test_kwargs = {"project_id": "test_id", "dialect": "standart"}
+        all_params = inspect.signature(pd.read_gbq).parameters
+        for param in all_params:
+            # 'query' - positional arg
+            if param not in test_kwargs and param != "query":
+                test_kwargs[param] = all_params[param].default
```

Comment:
maybe something like (untested)
```suggestion
        test_kwargs = inspect.signature(pd.read_gbq).parameters.copy()
        test_kwargs.update(project_id="test_id", dialect="standart")
        test_kwargs.pop("query", None)
```
?